src/cljs/main/broadfcui/page/workspace/monitor/workflow_details.cljs

```diff
@@ -62,6 +63,22 @@
          (for [[k v] (:data props)]
            [:div {} k [:span {:style {:margin "0 1em"}} "→"] (display-value v)])])])})
 
+;; display workflow-level and call-level failures
+(react/defc FailureMessages
+  {:get-initial-state
+   (fn []
+     {:expanded false})
+   :render
+   (fn [{:keys [props state]}]
+     [:div {}
+      (create-field
+        "Failures"
+        (style/create-link {:text (if (:expanded @state) "Hide" "Show")
+                            :onClick #(swap! state assoc :expanded (not (:expanded @state)))}))
+      (when (:expanded @state)
+        [:div {:style {:padding "0.25em 0 0.25em 1em"}}
```

Comment:
OK - this is copy/paste and I have no idea how em/rem work.  Is this what you want?
`{:padding "0.25rem 0 0.25rem 1rem"}`